common/ContractUtils.js

```diff
@@ -19,7 +19,7 @@ const revertWrapper = result => {
   if (isObject(result)) {
     // Iterate over the properties of the object and see if any match the revert value.
     for (let prop in result) {
-      if (result[prop].toString() === revertValue) {
+      if (result[prop] && result[prop].toString() === revertValue) {
```

Comment:
For some properties in a BN, this can be `null` which causes the `.toString()` to throw. We can assume that if the `result `is an Object, its OK to have property values equal to `null`, as long as they are not that revert magic number.